script/integration-test

```diff
@@ -44,7 +45,7 @@ do
     }
   }'
   echo "Filling in data to integration_test index"
-  curl -s -H "Content-Type: application/x-ndjson" -XPOST localhost:49200/_bulk --data-binary "@../documents.dat"; echo
+  curl -s -H "Content-Type: application/x-ndjson" -XPOST localhost:49200/_bulk --data-binary "@documents.dat"; echo
```

Comment:
What is `x-ndjson`? I've never heard of that.

Reply:
new line delimited json. The `.dat` files contain multiple objects separated by newlines. Common pattern afaik for using the bulk api like this. https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-bulk.html#bulk-curl